README.md

```diff
@@ -1,22 +1,28 @@
-# BEM hint [![Build Status](https://travis-ci.org/eGavr/bemhint.svg)](https://travis-ci.org/eGavr/bemhint) [![Coverage Status](https://img.shields.io/coveralls/eGavr/bemhint.svg)](https://coveralls.io/r/eGavr/bemhint?branch=master) [![Dependency Status](https://david-dm.org/eGavr/bemhint.svg)](https://david-dm.org/eGavr/bemhint) [![devDependency Status](https://david-dm.org/eGavr/bemhint/dev-status.svg)](https://david-dm.org/eGavr/bemhint#info=devDependencies)
+# BEM hint [![Build Status](https://travis-ci.org/bem/bemhint.svg)](https://travis-ci.org/bem/bemhint) [![Coverage Status](https://img.shields.io/coveralls/bem/bemhint.svg)](https://coveralls.io/r/bem/bemhint?branch=master) [![Dependency Status](https://david-dm.org/bem/bemhint.svg)](https://david-dm.org/bem/bemhint) [![devDependency Status](https://david-dm.org/bem/bemhint/dev-status.svg)](https://david-dm.org/bem/bemhint#info=devDependencies)
+
+**Can be used in projects based on [bem-bl](https://github.com/bem/bem-bl).**
```

Comment:
I had a task to create only such checking rule, so now the tool can be used only with `bem-bl`, probably some other rules will be implemented _tomorrow_ .